src/constants.js

```diff
@@ -0,0 +1,74 @@
+const PRIMARY = {
+  light: '#6f8aee',
+  base: '#5a79e6',
+  midDark: '#5973e7',
+  dark: '#4862e1',
+};
+
+const SECONDARY = {
+  light: '#ebeff5',
+  base: '#ebebeb',
+  midDark: '#ccd5f8',
+  dark: '#758092',
+};
+
+const TYPOGRAPHY = {
+  main: '#39393b',
+  heading: '#202020',
+};
+
+const RED = {
+  base: '#ff3131',
+};
+
+const GRAY = {
+  white: '#ffffff',
+  light: '#f0f2f5',
+  midLight: '#d6d6d6',
+  midDark: '#c2c8d0',
+  dark: '#757575',
+  black: '#000000',
+};
+
+export const palette = {
+  PRIMARY,
+  SECONDARY,
+  TYPOGRAPHY,
+  RED,
+  GRAY,
+};
+
+const FONT = {
+  main: 'Poppins',
+  heading: 'Poppins',
+};
+
```

Comment:
These items below (shadow, border etc.) are legacy constants. I'm not really a fan of keeping them in this file as they belong to specific components. If any, I'd use `SHADOW.primary`, `SHADOW.secondary` etc. convention.
What do you think?